example/src/main/java/org/nervos/ckb/ACPTransactionExample.java

```diff
@@ -0,0 +1,187 @@
+package org.nervos.ckb;
+
+import static org.nervos.ckb.utils.Const.*;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.nervos.ckb.address.Network;
+import org.nervos.ckb.crypto.secp256k1.Sign;
+import org.nervos.ckb.indexer.*;
+import org.nervos.ckb.service.Api;
+import org.nervos.ckb.transaction.*;
+import org.nervos.ckb.type.OutPoint;
+import org.nervos.ckb.type.Script;
+import org.nervos.ckb.type.Witness;
+import org.nervos.ckb.type.cell.CellDep;
+import org.nervos.ckb.type.cell.CellInput;
+import org.nervos.ckb.type.cell.CellOutput;
+import org.nervos.ckb.type.transaction.Transaction;
+import org.nervos.ckb.utils.Numeric;
+import org.nervos.ckb.utils.Utils;
+import org.nervos.ckb.utils.address.AddressGenerator;
+import org.nervos.ckb.utils.address.AddressParser;
+
+/** Copyright © 2021 Nervos Foundation. All rights reserved. */
+// ACP RFC:
+// https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0026-anyone-can-pay/0026-anyone-can-pay.md
+public class ACPTransactionExample {
+  // ACP_CKB_MINIMUM is set to 9, which means in each transaction, one must at least transfers 10^9
+  // shannons, or 10 CKBytes into the anyone-can-pay cel
+  private static final String ACP_CKB_MINIMUM = "09";
+
+  private static Api api;
+  private static CkbIndexerApi ckbIndexerApi;
+
+  private static final List<String> SendPrivateKeys =
+      Collections.singletonList("d00c06bfd800d27397002dca6fb0993d5ba6399b4238b2f29ee9deb97593d2bc");
+  private static final List<String> SendAddresses =
+      Collections.singletonList("ckt1qyqvsv5240xeh85wvnau2eky8pwrhh4jr8ts8vyj37");
+  private static final List<String> ReceiveAddresses =
+      Collections.singletonList("ckt1qyqtnz38fht9nvmrfdeunrhdtp29n0gagkps4duhek");
+  private static String receiverAcpAddress;
+
+  static {
+    api = new Api(NODE_URL, false);
+    ckbIndexerApi = new CkbIndexerApi(CKB_INDEXER_URL, true);
+
+    Script receiverScript = AddressParser.parse(ReceiveAddresses.get(0)).script;
+    receiverScript.codeHash = ACP_CODE_HASH;
+    receiverScript.args = receiverScript.args + ACP_CKB_MINIMUM;
+    receiverAcpAddress = AddressGenerator.generate(Network.TESTNET, receiverScript);
+  }
+
+  public static void main(String[] args) throws Exception {
+    System.out.println(
+        "Before transferring, first sender's balance: "
+            + getBalance(SendAddresses.get(0)).divide(UnitCKB).toString(10)
+            + " CKB");
+
+    String acpHash = createACPCell();
+    System.out.println("Create acp cell tx hash: " + acpHash);
+
+    // waiting transaction into block, sometimes you should wait more seconds
+    Thread.sleep(30000);
+    System.out.println("Transfer SUDT tx hash: " + transfer(new OutPoint(acpHash, "0x0")));
```

Comment:
there should be Transfer `CKB`?

Reply:
Updated https://github.com/nervosnetwork/ckb-sdk-java/pull/382/commits/fb6f3c18af1386b5a960b7f06af73f2a014d3d55